apps/style/common.scss

```diff
@@ -435,9 +443,17 @@ button.arrow:disabled {
   #codeWorkspace {
     left: $width;
   }
+  // specificially do not(.applab) so that we override the equivalent rule
+  // defined outside the media query
+  #codeWorkspace:not(.applab) {
```

Comment:
Yeah, it's probably not a great practice. I expect to be able to get rid of it when I address my TODO item about how to better share styling between components